src/Microsoft.PowerShell.Commands.Utility/commands/utility/select-object.cs

```diff
@@ -36,7 +36,7 @@ internal PSPropertyExpressionFilter(string[] wildcardPatternsStrings)
 
         /// <summary>
         /// try to match the expression against the array of wildcard patterns.
-        /// the first match shortcircuits the search
+        /// the first match shortcircuits the search.
```

Comment:
Uppercase T at the start of the sentence.

Reply:
Fixed.